### PR TITLE
add icons

### LINKED ISSUE #5: Configure better icons for bottom tabs
* [ ] Three tabs right now - Home, Captions, Settings. Suggest better icons for them.

### PR DESCRIPTION
**Pull request by ampsteric**
resolved #5.
please review and accept my PR and allow me to be a contributor for CapShun and Hacktoberfest as well.

best regards,
ampsteric

### CODE REVIEW

**Comment by devcer** [comment]
@ampsteric How about we use `<Ionicons name="md-list" ... />` for the captions screen. it is a list of captions anyway. so we don't have to do install material icons package for a single icon? Everything else looks good.

**Comment by ampsteric** [comment]
Okay so you are talking about this one rt?
md-list

Yeah , we can surely do that as it looks good too.
So should I make another PR?

**Comment by devcer** [comment]
yes. That one. You can push your changes to this same PR.

**Comment by ampsteric** [comment]
Done.
 please review and accept.

File: package.json [hunk 1]
```diff
@@ -12,13 +12,11 @@
   "jest": {
     "preset": "jest-expo"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn lint"
-    }
```
Comment (devcer) [inline package.json:18]: @ampsteric No need to add new packages. These scripts should not be deleted.

**Comment by ampsteric** [comment]
@devcer the above script was not allowing me to commit before making a PR.
can you please accept my PR and then add the script again into master or please tell me how to be able to commit with initiating the above script.

**Comment by devcer** [comment]
@ampsteric Ignore the warnings but if it is errors then you should fix them! this script performs a static code analysis on every PR before you commit. I can help you with fixing them if you see any errors.

**Comment by ampsteric** [comment]
@devcer I have added the above script.
Please review and accept my PR.

File: package.json (hunk 1 above)
Comment (devcer) [reply package.json:18]: @ampsteric can you take care of this?

**Comment by devcer** [comment]
@ampsteric can you look at my review comment? I think the packages you added to `package.json` are not required. You can remove them, test it and push your changes.

**Comment by ampsteric** [comment]
@devcer I have removed the unnecessary dependencies from package.json.
do let me know if there is anything else you want me to remove.
please review and accept my PR.

File: package.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
     "react-native-multiple-select": "^0.5.5",
     "react-native-safe-area-context": "~3.0.7",
     "react-native-screens": "~2.9.0",
+    "react-native-vector-icons": "^7.1.0",
```
Comment (devcer) [inline package.json:43]: @ampsteric You still haven't to remove this `react-native-vector-icons` package and delete package-lock.json as well(we use yarn, so we commit yarn.lock).

**Comment by ampsteric** [comment]
Okay. so I have removed the package that you mentioned and deleted package-lock.json as well.
Please review.

**Review by devcer — APPROVED** [review]

**Comment by ampsteric** [comment]
It is my humble request to the maintainers to please label my PR as "hacktoberfest-accepted" or mark the repo with the "hacktoberfest"  topic so that it can be counted as a valid PR for my first hacktoberfest.

best regards,
ampsteric

**Pull request merged**